v4.0.1/tasks-schema.json

```diff
@@ -0,0 +1,1555 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://raw.githubusercontent.com/hubverse-org/schemas/main/v4.0.1/tasks-schema.json",
+    "title": "Schema for Modeling Hub model task definitions",
+    "description": "This is the schema of the tasks.json configuration file that defines the tasks within a modeling hub.",
+    "type": "object",
+    "properties": {
+        "schema_version": {
+            "description": "URL to a version of the Modeling Hub schema tasks-schema.json file (see https://github.com/hubverse-org/schemas). Used to declare the schema version a 'tasks.json' file is written for and for config file validation. The URL provided should be the URL to the raw content of the schema file on GitHub.",
+            "examples": [
+                "https://raw.githubusercontent.com/hubverse-org/schemas/main/v3.0.0/tasks-schema.json"
+            ],
+            "type": "string",
+            "format": "uri"
+        },
+        "rounds": {
+            "description": "Array of modeling round properties",
+            "type": "array",
+            "items": {
+                "type": "object",
+                "description": "Individual modeling round properties",
+                "properties": {
+                    "round_id_from_variable": {
+                        "description": "Whether the round identifier is encoded by a task id variable in the data.",
+                        "type": "boolean"
+                    },
+                    "round_id": {
+                        "description": "Round identifier. If round_id_from_variable = true, round_id should be the name of a task id variable present in all sets of modeling task specifications",
+                        "examples": [
+                            "round_1",
```

Comment:
I noticed the example here was demonstrating an incorrect round ID format (`round-1` 🙈 ) so I fixed it